config/prod.secret.exs

```diff
@@ -27,6 +27,7 @@ config :auth_mvp, AuthMvpWeb.Endpoint,
   http: [:inet6, port: String.to_integer(System.get_env("PORT") || "4000")],
   secret_key_base: secret_key_base
 
+config :joken, default_signer: System.get_env("SECRET_KEY_BASE")
```

Comment:
Would be good to add `SECRET_KEY_BASE` environment variable to `.env_sample` file so people know they need it. 👍

Reply:
Added on the PR #10

Reply:
Thanks. 👍